letsencrypt/tests/cli_test.py

```diff
@@ -93,6 +94,29 @@ def test_help(self):
             from letsencrypt import cli
             self.assertTrue(cli.USAGE in out)
 
+    def test_configurator_selection(self):
+        real_plugins = disco.PluginsRegistry.find_all()
+        args = ['--agree-eula', '--apache', '--authenticator', 'standalone']
+
+        with mock.patch('letsencrypt.cli.plugins_testable') as plugins:
+            plugins.return_value = {"apache": True, "nginx": True}
+            ret, _, _, _ = self._call(args)
+            self.assertTrue("Too many flags setting" in ret)
+
+        if "nginx" in real_plugins:
```

Comment:
Can we please not do that at all? This code will not even be covered by our test suite, `nginx` will not be installed